client/src/App.test.js

```diff
@@ -1,9 +1,27 @@
 import React from 'react';
-import { render } from '@testing-library/react';
+import { render, screen, wait } from '@testing-library/react';
+import { FetchMock } from '@react-mock/fetch';
 import App from './App';
+import { findRenderedComponentWithType } from 'react-dom/test-utils';
 
-test('renders learn react link', () => {
-  const { getByText } = render(<App />);
-  const linkElement = getByText(/learn react/i);
-  expect(linkElement).toBeInTheDocument();
+const renderComponent = () =>
+render(
+    <FetchMock
+    options={
+        {matcher: '/api/v1/test-servlet', method: 'GET', response: "Hello world"}
+    }
+    >
+    <App />
```

Comment:
Can you indent <App /> so its more clear its a child of FetchMock

Reply:
Done